pkgs/development/libraries/mesa/default.nix

```diff
@@ -205,10 +211,13 @@ self = stdenv.mkDerivation {
     # meson auto_features enables these features, but we do not want them
     "-Dandroid-libbacktrace=disabled"
 
+    # from meson.build: Intel Ray Tracing is only supported on x86_64
+    (lib.mesonEnable "intel-rt" (stdenv.hostPlatform == "x86_64-linux"))
+
   ] ++ lib.optionals stdenv.isLinux [
-    "-Dglvnd=true"
+    "-Dglvnd=enabled"
```

Comment:
This will probably cause a merge conflict, since the code in Staging uses `lib.mesonBool`:

https://github.com/NixOS/nixpkgs/blob/1c4c369b07d2eaa9db33591d7916858a7dc95402/pkgs/development/libraries/mesa/default.nix#L244

Reply:
I changed it because of a build warning, I'll keep that in mind 👍🏻 

What's the difference between `lib.mesonBool` and `lib.mesonEnable` ?